scripts/source-maintenance.sh

```diff
@@ -68,6 +76,10 @@ if test "${ASVER}" != "2.04" ; then
 else
 	echo "Found astyle ${ASVER}. Formatting..."
 fi
+if test "${SkipAstyleVersionCheck}" = "yes" && test -z "${ASVER}" ; then
+    echo "Ignoring astyle version mismatch as instrcuted"
```

Comment:
Evidently, scripts/spell-check.sh does not catch this particular typo.

```suggestion
    echo "Ignoring astyle version mismatch due to --skip-astyle-version-check"
```